### ISSUE TITLE
Can not install btnx on Manjaro Linux

### ISSUE THREAD

**Issue by linhnd1303** [cdobrich/btnx-config#13]
I meet this error when tried to install btnx on Manjaro linux through AUR with command: yay -S btnx

:: Proceed with installation? [Y/n] 
(1/1) checking keys in keyring                                                                        [############################################################] 100%
(1/1) checking package integrity                                                                      [############################################################] 100%
(1/1) loading package files                                                                           [############################################################] 100%
(1/1) checking for file conflicts                                                                     [############################################################] 100%
error: failed to commit transaction (conflicting files)
btnx-config-git: /usr/sbin exists in filesystem (owned by filesystem)
Errors occurred, no packages were upgraded.

**Comment by cdobrich** [comment]
Hello. I am not familiar Manjaro or their package tool **yay**. I do not know who is maintaining the package of btnx& btnx-config in the Manjaro package repositories.

I am happy to try to help you compile it yourself. It's not complicated.

**Comment by linhnd1303** [comment]
Hello cdobrich, 
I have to say that I really appreciated this extremely useful btnx tool you've made. 

About the problem I reported above. I just did following steps and it worked. 

1. Step 1: install libglade through Pacman
![Screenshot_20201228_083340](https://user-images.githubusercontent.com/38790273/103183957-13660e00-48e8-11eb-9752-162ba1f8f50b.png)
2. Step 2: Download your btnx-master & btnx-config from github. 
3. Step 3: "./configure" then "make" then "sudo make install" 
4. Step 4: it's worked ^^

Thanks a lot for your time and efforts.
